scrapers/PuffyNetworkSite.yml

```
@@ -0,0 +1,36 @@
+name: PuffyNetworkSite
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - simplyanal.com
+      - weliketosuck.com
+      - wetandpissy.com
+      - wetandpuffy.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="header-title"]/h2
+        postProcess:
+          - replace:
+            - regex: ^(.*)[—]\s+
+              with:
+      Date:
+        selector: //span[@class="date-movie"]/b
+        postProcess:
+          - parseDate: Jan 02, 2006
```

```suggestion
          - parseDate: Jan 2, 2006
```
`https://www.wetandpuffy.com/videos/jane-toy/`
`https://www.wetandpissy.com/videos/video-nurse-naomi/`
The 0 breaks the parsing for date in most scenes. Was there a case where 0 was needed?